.devcontainer/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM mcr.microsoft.com/devcontainers/go:1-1.21-bookworm
+
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends postgresql-client
```

Comment:
Do we want some other tools in here like `jq`, `golangci-lint`, and `python3.11`?
I use a pile of node based linting tools as well so maybe we add them?
Not saying to add all this stuff but here is my Fedora dev box guide as an example. https://github.com/xbcsmith/xbcsmith/blob/main/docs/fedora_38_dev_box.md

Reply:
To my surprise, this image already includes python (have to use `python3` explicitly). Let me know if there are any additional tools we should add.

Reply:
Lets start here